### DISCUSSION TITLE
Customizing the text "MY BLOG" ?

### DISCUSSION THREAD

**Discussion by sgichuki** [lxndrblz/anatole#321] [Q&A] [answered]
I tried out your theme, like the simple and clean layout but I was wondering if it is possible to customize the  text under the main profile picture i.e. "MY BLOG" in order for it to say something else or even be in another case. I have looked through the ReadMe file and cannot see anything related to this. Am I missing something? Please let me know, thanks.

**Comment by lxndrblz** [comment, marked as answer]
Please check the wiki. 

https://github.com/lxndrblz/anatole/wiki/1%EF%B8%8F%E2%83%A3-Essential-Steps#first-steps

It should answer all of your questions.
